### PR TITLE
Send "urgent tickets" CSVs

### LINKED ISSUE #50: Update cyhy-mailer to only send 2 new CSVs based on urgency
## Task ##

In support of BOD 22-01 requirement 11: Update [cyhy-mailer](https://github.com/cisagov/cyhy-mailer/blob/develop/cyhy/mailer/CybexMessage.py#L75-L155) to only send the 2 new CSVs created in #49.

## Comments ##

The associated CSVs are generated here: https://github.com/cisagov/cyhy-reports/blob/develop/extras/create_snapshots_reports_scorecard.py#L747-L777 

### References ###

- Epic https://github.com/cisagov/cool-system-internal/issues/107
- Story #34 
- Current Estimated LOE: 3 hours for this task.

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##

This PR modifies `cyhy-mailer` to attach the [new "urgent tickets" CSVs](https://github.com/cisagov/cyhy-reports/pull/70) (instead of the previous "critical" and "high" CSVs) when emailing the Cyber Exposure (CybEx) scorecard.

While I was here, I updated the pre-commit config and a couple of related changes (see commits https://github.com/cisagov/cyhy-mailer/pull/92/commits/8fef4c7ac5f24964daf972e8b956f8f31ac26204, https://github.com/cisagov/cyhy-mailer/pull/92/commits/fc34ca5fb4d1e47ee0eebaf520a31da5bee4ce29, https://github.com/cisagov/cyhy-mailer/pull/92/commits/d396e61b9a4c06fc246e124ffcc50cdd42468917).

## 💭 Motivation and context ##

* **This PR is dependent on the changes in https://github.com/cisagov/cyhy-reports/pull/72 and https://github.com/cisagov/ncats-webd/pull/27 and is blocked until they are deployed**
* This PR resolves https://github.com/cisagov/cyhy-system/issues/50
* This PR covers part of the work requested in https://github.com/cisagov/cyhy-system/issues/34

## 🧪 Testing ##

I tested this change by temporarily modifying the default email/BCC addresses to point to my email address, then ran this command:
```
$ AWS_DEFAULT_REGION=us-east-1 AWS_PROFILE=my_email_sending_profile cyhy-mailer cybex \
--cybex-scorecard-dir=test_scorecards/ --db-creds=./secrets/db-creds.yml --summary-to=me@myemail.gov
```
After I ran that, I confirmed that I received an email containing the new "urgent tickets" CSVs (that I had previously generated while testing https://github.com/cisagov/cyhy-reports/pull/70).

I also verified that there were no failures when I ran `pytest`.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Confirm that the [cyhy-reports `epic/bod-22-01` branch](https://github.com/cisagov/cyhy-reports/tree/epic/bod-22-01) (which includes https://github.com/cisagov/cyhy-reports/pull/70) has been deployed or will be deployed before this PR.

## ✅ Post-merge checklist ##

- [x] Add a tag (but don't create a release).

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -22,55 +28,107 @@ repos:
...
+    hooks:
+      - id: black
   - repo: https://gitlab.com/pycqa/flake8
-    rev: 3.9.1
+    rev: 3.9.2
     hooks:
       - id: flake8
         additional_dependencies:
           - flake8-docstrings
+  - repo: https://github.com/PyCQA/isort
```
Comment (mcdonnnj) [inline .pre-commit-config.yaml:100]: Please add our `.isort.cfg` file. I believe `pre-commit` should be run once this is in place to get everything aligned. You might want to add `known_first_party=mongo_db_from_config` to `.isort.cfg` to group that package of ours in with the other first party imports.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -22,55 +28,107 @@ repos:
...
-        # GitHub's autorecognition fu to work, so we should leave it
-        # alone.
-        exclude: LICENSE.md
+        args:
+          - --config=.mdl_config.yaml
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v2.6.1
+    hooks:
+      - id: prettier
   - repo: https://github.com/adrienverge/yamllint
```
Comment (mcdonnnj) [inline .pre-commit-config.yaml:43]: Would you update `.yamllint` just for consistency.

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -22,55 +28,107 @@ repos:
...
+    rev: v0.31.1
     hooks:
       - id: markdownlint
-        # The LICENSE.md must match the license text exactly for
-        # GitHub's autorecognition fu to work, so we should leave it
-        # alone.
-        exclude: LICENSE.md
+        args:
+          - --config=.mdl_config.yaml
+  - repo: https://github.com/pre-commit/mirrors-prettier
```
Comment (mcdonnnj) [inline .pre-commit-config.yaml:39]: Would you update `.prettierignore` just for consistency.

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Everything functionality wise looks good. On the back of my comments please make sure to run a `pre-commit run --all-files` to make sure the updated `pre-commit` configuration is applied to this repository since there is no GitHub Actions testing.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (dav3r) [reply .pre-commit-config.yaml:100]: Good call - please see https://github.com/cisagov/cyhy-mailer/pull/92/commits/d0fe8c5be75b3dcca595ddb987c5645b5ebafab1.

File: .pre-commit-config.yaml (hunk 2 above)
Comment (dav3r) [reply .pre-commit-config.yaml:43]: 👍  Please see https://github.com/cisagov/cyhy-mailer/pull/92/commits/682b9ffe0c399c9684100c5fc03b1ea2b3df0897.

File: .pre-commit-config.yaml (hunk 3 above)
Comment (dav3r) [reply .pre-commit-config.yaml:39]: 👍  Please see https://github.com/cisagov/cyhy-mailer/pull/92/commits/682b9ffe0c399c9684100c5fc03b1ea2b3df0897.

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔

**Review by felddy — APPROVED** [review]
"So Urgent!" 🎷

**Pull request merged**